### ISSUE TITLE
Cannot read property 'renderedLength' of undefined

### ISSUE THREAD

**Issue by evdama** [btd/rollup-plugin-visualizer#63]
My bundles html page doesn't show any anything anymore (blank page in Chrome). I'm getting this error in my chrome dev tools console:

```
edm-bundles.html:3180 Uncaught TypeError: Cannot read property 'renderedLength' of undefined
    at edm-bundles.html:3180
    at Node.node_eachAfter [as eachAfter] (edm-bundles.html:1142)
    at drawChart (edm-bundles.html:3173)
    at HTMLDocument.<anonymous> (edm-bundles.html:3356)
```

**Comment by btd** [comment]
Can yoy share your html file?

**Comment by evdama** [comment]
sure https://gist.github.com/2cf946a5661a2f1f7474370ed1462c8f

**Comment by btd** [comment]
May i ask you debug it a little? It looks currently rollup generate chunk but d3 in some reason make it look like a leaf in hierarchy.
Would you mind check size of failed chunk? For shared file it is `client.1f4ca05e.js`, rollup reports that this file does not contain any anything, but it looks strange at least. Could you also add which rollup version you are using?

**Comment by evdama** [comment]
I'm using the latest rollup version, 2.0.6.

`client.1f4ca05e.js` contains `import"./client.3b1904fc.js";` which in turn all the code https://gist.github.com/44a02df8fe7be97d4cc4a055b0c572f3

**Comment by btd** [comment]
I just published 3.3.2, could you try?
I still not sure why rollup report it as empty chunk without deps if it contains import to another chunk.

**Comment by evdama** [comment]
Thanks much, it's working again :)
